index.bs

```diff
@@ -521,19 +527,23 @@ When this method is invoked, the user agent MUST execute the following algorithm
     [=relevant settings object=]. If |callerOrigin| is an [=opaque origin=], return a {{DOMException}} whose name is
     "{{NotAllowedError}}", and terminate this algorithm.
 
-1. If the {{PublicKeyCredentialEntity/id}} member of |options|.{{MakeCredentialOptions/rp}} is [=present|not present=], then set
-    |rpId| to |callerOrigin|.
+1. Let |effectiveDomain| be the |callerOrigin|'s [=effective domain=]. If |effectiveDomain| is null, then return a
```

Comment:
ok. Tho note that we /are/ going to have normative deltas from WD-05 given the latter was a rush job with known normative issues outstanding....